src/client/kv_client.rs

```diff
@@ -284,13 +299,40 @@ impl KvClient {
         match result {
             Ok(mut r) => {
                 let content = if let Some(body) = r.0.body() {
-                    match serde_json::from_slice(body.as_ref()) {
-                        Ok(v) => Some(v),
-                        Err(e) => {
-                            return Err(ClientError::RequestFailed {
-                                reason: Some(e.to_string()),
-                                key: r.1,
-                            });
+                    // Leading 0 indicates response to subdoc multi lookup
+                    if body[0] == 0 {
```

Comment:
Can we do this off op_code from the result?

Reply:
Done